File: Server/src/main/java/com/api/server/foodb/PfamMembership.java
package com.api.server.foodb;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "pfam_memberships")
public class PfamMembership {
    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "enzyme_id")
    private Integer enzymeId;

    @Column(name = "pfam_id")
    private Integer pfamId;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/PfamMembership.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("pfam_memberships")]
[Index("EnzymeId", "PfamId", Name = "index_pfam_memberships_on_enzyme_id_and_pfam_id", IsUnique = true)]
public class PfamMembership
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("enzyme_id")] public int? EnzymeId { get; set; }

    [Column("pfam_id")] public int? PfamId { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }
}